### ISSUE TITLE
Upgrade Django re security issue CVE-2019-12308

### ISSUE THREAD

**Issue by gregelin** [GovReady/govready-q#634]
We are currently ignoring Django update 2.1.2, but there is a security update for Django in 2.1.3 that addresses vulnerability.

@JoshData What is the reason that we are ignoring Django updates? Can we move/test Django 2.1.3?

CVE-2019-12308 - https://nvd.nist.gov/vuln/detail/CVE-2019-12308

moderate severity
Vulnerable versions: >= 2.1.0, < 2.1.9
Patched version: 2.1.9
An issue was discovered in Django 1.11 before 1.11.21, 2.1 before 2.1.9, and 2.2 before 2.2.2. The clickable Current URL value displayed by the AdminURLFieldWidget displays the provided value without validating it as a safe URL. Thus, an unvalidated value stored in the database, or a value provided as a URL query parameter payload, could result in an clickable JavaScript link.

`requirements_txt_checker_ignoreupdates.txt`
```
setuptools==40.2.0
django-notifications-hq==1.5.0
Django==2.1.2
```

**Comment by JoshData** [comment]
I don't recall any specific reason why we're holding Django back. It may be that there were no changes that were relevant to us, so it wasn't worth the risk of accidental breakage. This vulnerability probably doesn't affect Q either because I don't think we have any models that show URL fields in the admin.

**Comment by gregelin** [comment]
@JoshData Thank you.

**Comment by gregelin** [comment]
With update of Django to 2.2.x, this issue can now be tested and addressed.

**Comment by gregelin** [comment]
Fixed in PR #798
